t/10-jobs.t

```diff
@@ -396,10 +399,17 @@ subtest 'carry over, including soft-fails' => sub {
     $job->discard_changes;
     is($job->result, OpenQA::Jobs::Constants::NONE, 'result is not yet set');
     is($job->comments, 0, 'no comment');
-    $job->done;
-    $job->discard_changes;
-    is($job->result, OpenQA::Jobs::Constants::FAILED, 'job result is failed');
-    is($job->comments, 0, 'no takeover');
+
+    subtest 'additional investigation notes provided on new failed' => sub {
+        copy('t/data/last_good.json', catfile(($job->_previous_scenario_jobs)[0]->result_dir(), 'vars.json'));
+        copy('t/data/first_bad.json', catfile($job->result_dir(), 'vars.json'));
```

Comment:
I thought you liked `Mojo::File`? 😉

Reply:
that must have been somone else ;) Here I don't see how it would help. Let's keep it as is if you don't mind

Reply:
```perl
path(($job->_previous_scenario_jobs)[0]->result_dir(), 'vars.json')->copy_to('t/data/last_good.json');
```

Reply:
🤷‍♀

Reply:
that looks to be the wrong way around. I will look into this for the next PR.